ush/plotObsMon

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# plotObsMon
+# parse input yaml plot file, apply to DA monitor templates to
+# create eva yaml files and run eva.
+
+import argparse
+from logger import Logger
+
+import os
+from re import sub
+import yaml
+from wxflow import parse_j2yaml, save_as_yaml
+from wxflow import add_to_datetime, to_timedelta, to_datetime
+
+
+def genYAML(input_yaml, output_yaml, config):
+    # read in YAML/Jinja template
+    final_config = parse_j2yaml(input_yaml, config)
+    save_as_yaml(final_config, output_yaml)
+
+#--------------------------------------------------------------------------------------------
+
+def camel_case(s):
+  s = sub(r"(_|-)+", " ", s).title().replace(" ", "")
+  return ''.join([s[0].lower(), s[1:]])
+
+#--------------------------------------------------------------------------------------------
+
+def loadConfig(satname, instrument, plot, cycle_tm, cycle_interval):
+    config = {}
+    config['SAT'] = satname
+    config['SENSOR'] = instrument
+    config['LEVELS'] = plot.get('levels')
+    config['RUN'] = plot.get('run')
```

Comment:
It might be easier to read/add extra things if doing something like this:
```
config = {
   'SAT': satname,
  'SENSOR': instrument,
...
}
```
Then you can still do:
`config['PDATE'] = cycle_tm` down below

Reply:
Done.